aggsender/types/types.go

```diff
@@ -131,7 +131,7 @@ type CertificateMetadata struct {
 }
 
 // NewCertificateMetadataFromHash returns a new CertificateMetadata from the given hash
-func NewCertificateMetadata(fromBlock, toBlock, createdAt uint64) *CertificateMetadata {
+func NewCertificateMetadata(toBlock, fromBlock, createdAt uint64) *CertificateMetadata {
```

Comment:
IMO it is more intuitive to provide the `fromBlock` as the first parameter (although we are encoding the `toBlock` first). WDYT?

```suggestion
func NewCertificateMetadata(fromBlock, toBlock, createdAt uint64) *CertificateMetadata {
```